tests/testthat.R

```diff
@@ -1,4 +1,16 @@
-library("testthat")
-library("vdiffr")
 
-test_check("vdiffr")
+mock_path <- file.path("testthat", "mock-pkg", "tests")
+cases_ver <- vdiffr:::cases_freetype_version(mock_path)
+system_ver <- vdiffr:::system_freetype_version()
+
+if (cases_ver != system_ver) {
```

Comment:
This is to avoid testing the results of checking the mock package. Tests will skip inside the mock package but then they won't correspond to what's expected in vdiffr's own tests.